README.md

```diff
@@ -603,6 +604,47 @@ async def evaluate(folder_of_litqa_v2_papers: str | os.PathLike) -> None:
     print(metrics_callback.eval_means)
 ```
 
+### Using Clients Directly
+
+One of the most powerful features of PaperQA2 is its ability to combine data from multiple metadata sources. For example, [Unpaywall](https://unpaywall.org/) can provide open access status/direct links to PDFs, [Crossref](https://www.crossref.org/) can provide bibtex, and [Semantic Scholar](https://www.semanticscholar.org/) can provide citation licenses. Here's a short demo of how to do this:
+
+```python
+from paperqa.clients import DocMetadataClient, ALL_CLIENTS
+
+client = DocMetadataClient(clients=ALL_CLIENTS)
+details = await client.query(title="Augmenting language models with chemistry tools")
+
+print(details.formatted_citation)
+# Andres M. Bran, Sam Cox, Oliver Schilter, Carlo Baldassari, Andrew D. White, and Philippe Schwaller.
+#  Augmenting large language models with chemistry tools. Nature Machine Intelligence,
+# 6:525-535, May 2024. URL: https://doi.org/10.1038/s42256-024-00832-8,
+# doi:10.1038/s42256-024-00832-8.
+# This article has 243 citations and is from a domain leading peer-reviewed journal.
```

Comment:
This should align with the citation count below— maybe we had a regression w recent changes. Or is it a copy-paste issue?

Reply:
I copy pasted because README's aren't rendered with wordwrap in code

Reply:
Oh I misunderstood your question! No I manually edited it. This was a timeout issue - don't remember which but one of the clients timed out and the other sent back 20 so I fetched from google scholar

Reply:
fixed